styles.css

```diff
@@ -132,13 +146,19 @@
   color: #1c1a19;
 }
 
+.work-detail {
+  margin-top: -124px;
+}
+
 .img-fluid {
+  display: block;
   width: 327px;
   height: 237px;
   margin-left: 24px;
   margin-right: 24px;
   column-gap: 80px;
-  margin-top: 104px;
+  margin-top: -14px;
+  margin-bottom: -44px;
 }
 
 .img-fluid2 {
```

Comment:
- [ ] While you made significant progress fixing your mobile version CSS, there is still a horizontal scroll, probably due to this image width. Kindly fix that.
Your design:
![image](https://user-images.githubusercontent.com/61919617/178023505-2811bcdd-ed57-4cca-afc2-af3539015c86.png)